test/bootstrap_checkbox_test.rb

```diff
@@ -477,63 +477,6 @@ class BootstrapCheckboxTest < ActionView::TestCase
                                                                     :street, checked: collection)
   end
 
```

Comment:
So the whole "custom" thing is gone? That makes life easier.

Reply:
No kidding, much easier. [Here's the relevant section](https://v5.getbootstrap.com/docs/5.0/migration/#forms) of the Bootstrap 5 migration docs. They're clearly incomplete and missing significant parts, but the linked section says about as much - the custom stuff is gone.